imports/server/schemas/DocumentWatch.ts

```diff
@@ -0,0 +1,32 @@
+import * as t from 'io-ts';
+import { date } from 'io-ts-types';
+import { Id } from '../../lib/schemas/regexes';
+import { Overrides, buildSchema } from '../../lib/schemas/typedSchemas';
+
+export const DocumentWatchCodec = t.type({
+  _id: t.string,
+  document: t.string,
+  watchId: t.string,
```

Comment:
I'm a little confused why we need both `watchId` and `_id` as separate fields here -- they seem to have the same lifetime and lifecycle, and last I checked I thought you could pass an explicit `_id` to `insert()` if you wanted to pick the ID in advance with `Random.id()`?

Reply:
They don't actually have the same lifecycle today - we update the `watchId` every time we refresh the watch, but the database document remains the same (i.e. the `_id` persists)